services/monitoring/docker-compose.yml.j2

```diff
@@ -313,8 +331,10 @@ services:
       resources:
         limits:
           memory: 128M
+          cpus: "0.5"
```

Comment:
might want to keep an eye on the exporters if they get too slow or accumulate data if limited too much.
But I agree on having them low

Reply:
Agreed